spec/requests/v1/sessions_spec.rb

```diff
@@ -113,19 +119,25 @@
         }
       end
 
-      it "refreshs the jwt token" do
+      it "refreshes the jwt token" do
         sleep(1)
-        expect { post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: { "Cookie" => response['Set-Cookie'] } }.to change { response.cookies.with_indifferent_access[:jwt] }
-        expect(jwt_cookie).to be_truthy
+        expect { post hyku_api.v1_tenant_users_refresh_path(tenant_id: account.tenant), headers: auth_header }.to change { response.cookies.with_indifferent_access[:jwt] }
```

Comment:
Is this test specific enough?  I suspect that removing the cookie is still changing it so would pass this test.  That might be why I had the truthy test as well.